README.dev.md

```diff
@@ -2,32 +2,61 @@
 Installing the Zulip Development environment
 ============================================
 
-You will need a machine with at least 2GB of RAM available (see
+Currently, you will need a machine with at least 2GB of available RAM (see
 https://github.com/zulip/zulip/issues/32 for a plan for how to
 dramatically reduce this requirement).
 
-Start by cloning this repository: `git clone https://github.com/zulip/zulip.git`
+Start by cloning the Zulip repository (commands below).
+
+For non-Windows host platforms, enter the following command in the terminal:
+`git clone https://github.com/zulip/zulip.git`
+
+For Windows host platforms, enter the following command in the terminal (the
+environment runs on Linux, so "-c core.autocrlf=false" avoids adding
+Windows line endings, which cause weird errors):
+`git clone https://github.com/zulip/zulip.git -c core.autocrlf=false`
+
```

Comment:
I think it'd be good to give this more of a heading-type field, e.g. 

<b>non-Windows</b>:
`command1`

**Windows**: 
`command2`